presto-docs/src/main/sphinx/sql/delete.rst

```diff
@@ -30,3 +30,8 @@ Delete all line items for low priority orders::
 Delete all orders::
 
     DELETE FROM orders;
+
+Limitations
+-------------
+Only :doc:`../connector/hive` currently supports DELETE and it works for the queries where one or more partitions are
```

Comment:
We should remove this entirely as this is the SQL documentation, not the Hive connector documentation.

Perhaps a more generic note like "some connectors may not support this operation or have restricted support, see the connector documentation for details".

Or should we add connector specific docs to all the various DDL and DML statements? Not sure how I feel about that.